persistence/persistence-mybatis/src/main/resources/org/cbioportal/persistence/mybatis/GenePanelMapper.xml

```diff
@@ -5,6 +5,7 @@
     <cache/>
 
    <resultMap id="genePanelResultMap" type="org.cbioportal.model.GenePanel">
+        <result property="internalId" column="internalId"/>       
```

Comment:
does this get presented to the user through the api? I think I remember controller level mappings/mixins which suppress this, but don't remember.